.travis.yml

```diff
@@ -1,9 +1,21 @@
 script:
-- |
-  docker build --target test --tag todo-app:test .
-  docker run todo-app:test tests/test_view_model.py
-  docker run todo-app:test tests/test_integration.py
-  docker run --env TRELLO_TOKEN=$TRELLO_TOKEN \
-             --env TRELLO_KEY=$TRELLO_KEY \
-             --env TRELLO_BOARD_NAME=MyTodos \
-             todo-app:test tests/test_selenium.py
+- docker build --target test --tag todo-app:test .
+- docker run todo-app:test tests/test_view_model.py
+- docker run todo-app:test tests/test_integration.py
+- docker run --env TRELLO_TOKEN=$TRELLO_TOKEN --env TRELLO_KEY=$TRELLO_KEY --env TRELLO_BOARD_NAME=MyTodos todo-app:test tests/test_selenium.py
+- echo $DOCKER_PASSWORD | docker login --username colinbudd --password-stdin
+- IMAGE_TAG="colinbudd/todo:$TRAVIS_COMMIT"
+- docker build --target production --tag $IMAGE_TAG . 
+- docker image tag $IMAGE_TAG colinbudd/todo:latest
+
+after_success:
```

Comment:
Note that this runs on every build - you'll deploy a new version of your app for every commit that is pushed. In a team of several developers that's going to make for a very unstable system, even if this isn't your production environment! We should ensure that this is only pushed for the main branch
You might want to look at the [Travis "deploy" lifecycle stage](https://docs.travis-ci.com/user/deployment/script/) - we can then easily ensure that this runs only on specified branches, and only for branch builds rather than PRs